src/tools/recipe2plan-cli.ts

```diff
@@ -69,12 +66,23 @@ const outFormat = (() => {
 
 void Flags.withDefaultReferenceMode(async () => {
   try {
-    Runtime.init('../..');
     fs.mkdirSync(opts.outdir, {recursive: true});
 
-    const manifest = await Runtime.parseFile(opts._[0]);
+    const loader = new Loader({});
+    const memoryProvider = new SimpleVolatileMemoryProvider();
+
+    const [target, ...rest] = opts._;
+    const targetFile = await loader.loadFile(target);
+    const toImport = rest
+      .map(arcsFile => path.relative(path.dirname(target), arcsFile))
+      .filter(arcsFile => !targetFile.includes(path.basename(arcsFile)))
+      .map(arcsFile => `import '${arcsFile}'`);
+
+    const bufferManifest = toImport.join('\n') + `\n${targetFile}`;
```

Comment:
As a principle we should not be rewriting the textual representation of the manifest like this. Instead, it would be preferable to take a step back and understand the need that we're trying to address and structure the code processing these manifests appropriately.

E.g.

We could have a _main_ manifest, for which we're producing plans, and extra set of _ancillary_ manifests, that provide us with context such as: is anyone reading data being produced in the _main_ manifest.

Concretely, instead of rewriting the main manifest by adding imports, maybe we can parse all the manifests and pass them to recipe2plan, so that recipe2plan can process them as they are.

We can talk more later today.